src/main/java/de/rondiplomatico/nds/NDSTile.java

```diff
@@ -108,6 +121,256 @@ public NDSTile(int level, WGS84Coordinate coord) {
         this(level, new NDSCoordinate(coord.getLongitude(), coord.getLatitude()));
     }
 
+    /**
+     * Get tile indices in x / y from id and level.
+     */
+    public int[] getTileXY() {

```

Comment:
This provides a straightforward way of retrieving the tile x-/y-indices, however, this can also be done, e.g., via `getTileXYfromTileNumber`. Thus, removed for conciseness.